presets/react/storybook/ssr/webpack.config.js

```diff
@@ -0,0 +1,162 @@
+const path = require('path');
+const sass = require('sass');
+const sassFunctions = require('sass-functions');
+const { DefinePlugin, HotModuleReplacementPlugin } = require('webpack');
+const babelConfig = require('../../babel.config');
+
+const CONFIG_NAME_CLIENT = 'client';
+const CONFIG_NAME_SERVER = 'server';
+const COMPILE_INDEX_SERVER = 0;
+const COMPILE_INDEX_CLIENT = 1;
+const ROOT_PKG_PATH = path.resolve(path.join(__dirname, '../../../../'));
+
+const fileLoaderRules = [{
```

Comment:
rewriting this feels super iffy, wouldn't it be possible to import webpack config from storybook and mutate the parts that we need?